lib/propolis/src/hw/uart/uart16550.rs

```diff
@@ -76,10 +92,22 @@ impl Uart {
             _ => {
                 panic!();
```

Comment:
Worth refactoring this a little bit so we also get the probe in this branch too? Ideally we don't panic but if we do, would be nice to know what caused it.

Reply:
Sure. Rather than refactoring to an option or something, I added the probe to that branch too -- assuming whoever is debugging is already tracing this probe, I figured it would be better to have a dummy value for the read byte there.